cli/tests/test_split_command.rs

```diff
@@ -263,8 +264,9 @@ fn test_split_with_default_description() {
     "###);
     assert!(!test_env.env_root().join("editor2").exists());
     insta::assert_snapshot!(get_log_output(&test_env, &workspace_path), @r###"
-    @  kkmpptxzrspx false test_bookmark
+    @  kkmpptxzrspx false
     ○  qpvuntsmwlqt false TESTED=TODO
+    │   test_bookmark
```

Comment:
Why does the bookmark end up rendered on the second line? Oh, because it's after the description, and the description has a newline. It looks a bit strange. Do you think we should put bookmarks before the description in the `get_log_output ()` template?

Reply:
Yeah, it's not a bad idea to change the template. I'll push another commit that does that.

RE: approval, it's not clear to me if @ilyagr has a strong objection to this change. Let me know if we need more discussion before submitting.

Reply:
It seems like we need to have a bit more discussion about this functionality change, so I moved the new tests in the base commit into a new PR: https://github.com/jj-vcs/jj/pull/5638